sky/execution.py

```diff
@@ -705,8 +705,9 @@ def _maybe_translate_local_file_mounts_and_sync_up(
     has_local_source_paths = (task.workdir is not None) or copy_mounts
     if has_local_source_paths:
         logger.info(
-            f'{colorama.Fore.YELLOW}Translating file_mounts with local '
-            f'source paths to SkyPilot Storage...{colorama.Style.RESET_ALL}')
+            f'{colorama.Fore.YELLOW}Translating workdir and file_mounts with '
```

Comment:
While we're at this, can we make this message be representative of what is being synced:
* If no workdir exists, but file_mounts exist: `Translating file_mounts`
* If workdir exists, but no file_mounts exist: `Translating workdir`
* If both workdir and file_mounts exist: `Translating workdir and file_mounts`